crates/cargo-wdk/src/actions/mod.rs

```diff
@@ -61,8 +61,29 @@ impl Profile {
 
 /// `TargetArch` for the action layer
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub enum TargetArch {
-    X64,
+pub enum CpuArchitecture {
```

Comment:
If we're creating an enum with the exact same name and variants as the one in wdk_build, I'd suggest we just use that instead.

Reply:
There shouldn't need to be a new enum here. It should just use the one from `wdk_build`. If the WDK normally uses x64 and ARM64 as the externally facing names for the variants, the tool should probably match that. IIRC clap has ways to convert if the strings don't match exactly. 

There is also the alternative of aliasing the names to accept different names for the same thing

Reply:
We did want to use the existing enum in `wdk_build` but weren't sure about the variant names and also whether it would be okay to add Clap's alias attributes on that enum (in case we needed to). Therefore we wanted to discuss this with you and the DDX team before proceeding.

Reply:
We had a chat with the DDX folks and they suggested using `Amd64` and `Arm64` which correspond exactly to the names used in the `wdk_build` enum. So let's just use that enum. Also let's expose the same names at the command line for now, which means no clap name/alias attributes are needed on this type.